deployment/helm/templates/agent.yaml

```diff
@@ -61,8 +61,6 @@ spec:
             mountPath: /host/var/run/dockershim.sock
           - name: devices
             mountPath: /run/udev
-        securityContext:
-          privileged: true
```

Comment:
is there any reason to leave this in with configurable parameter in values.yaml (that defaults to false)?

Reply:
I don't see a reason right now to have it privileged. But i could do the same thing i do in [this pr](https://github.com/deislabs/akri/pull/148) and make the security context configurable.

Reply:
a simplified version of that pr is what i was suggesting.